src/ios/Screenshot.m

```diff
@@ -72,7 +89,6 @@ - (void)saveScreenshot:(CDVInvokedUrlCommand*)command
     contextInfo.callbackId = callbackId;
     contextInfo.jpgPath = jpgPath;
 
-	UIImageWriteToSavedPhotosAlbum(image, self, @selector(image:didFinishSaving:contextInfo:), (__bridge_retained void *)contextInfo);
     [imageData writeToFile:jpgPath atomically:NO];
```

Comment:
I know this is outside of the scope of the PR, but it is related: Maybe we can change `automatically` to `TRUE` here. This is what the docs say about it: `If YES, the data is written to a backup file, and then—assuming no errors occur—the backup file is renamed to the name specified by path; otherwise, the data is written directly to path.`. So it seems like `YES` would be safer.